tests/framework-tests/core/js/DataBindingTests.js

```diff
@@ -2297,6 +2461,6 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         return togo;
     };
 
-    fluid.test.runTests(["fluid.tests.fluid5659root"]);
+    // fluid.test.runTests(["fluid.tests.fluid5659root"]);
```

Comment:
Oops... I often comment out that line when debugging since even when disabled, IoC framework tests generate so much "construction junk" at startup ...